BGAnimations/ScreenSelectMusic overlay/SongDescription/SongDescription.lua

```diff
@@ -49,6 +49,13 @@ af[#af+1] = Def.ActorFrame{
 				local song = GAMESTATE:GetCurrentSong()
 				self:settext( song and song:GetDisplayArtist() or "" )
 			end
+
+			local maxwidth = _w - 60
+			local bubble = self:GetParent():GetParent():GetChild("Bubble")
+			if bubble and bubble:GetVisible() then
```

Comment:
I'm not quite sure on the order of evaluation here. Does bubble get unhidden first or does this statement get evaluated. I think it might be better to just check `if song and (song:IsLong() or song:IsMarathon())` and not have to worry about it.

Also maybe(?) this should probably be moved to the else statement above. I'm not sure if the bubble shows up in course mode.

Reply:
Interestingly he bubble does not show up in course mode. I'm not sure how the credits work for courses.

I changed the code like you suggested!